# __init__.py
# ruff: noqa: F401

from grpc import ServicerContext as GrpcContext

from leapfrogai_sdk.audio.audio_pb2 import (
    AudioMetadata,
    AudioRequest,
    AudioResponse,
)
from leapfrogai_sdk.audio.audio_pb2_grpc import Audio, AudioServicer, AudioStub
from leapfrogai_sdk.chat.chat_pb2 import (
    ChatCompletionChoice,
    ChatCompletionRequest,
    ChatCompletionResponse,
    ChatItem,
    ChatRole,
)
from leapfrogai_sdk.chat.chat_pb2_grpc import (
    ChatCompletionService,
    ChatCompletionServiceServicer,
    ChatCompletionServiceStub,
    ChatCompletionStreamService,
    ChatCompletionStreamServiceServicer,
    ChatCompletionStreamServiceStub,
)
from leapfrogai_sdk.completion.completion_pb2 import (
    CompletionChoice,
    CompletionFinishReason,
    CompletionRequest,
    CompletionResponse,
    CompletionUsage,
)
from leapfrogai_sdk.completion.completion_pb2_grpc import (
    CompletionService,
    CompletionServiceServicer,
    CompletionServiceStub,
    CompletionStreamService,
    CompletionStreamServiceServicer,
    CompletionStreamServiceStub,
)
from leapfrogai_sdk.config import BackendConfig
from leapfrogai_sdk.embeddings.embeddings_pb2 import (
    Embedding,
    EmbeddingRequest,
    EmbeddingResponse,
)
from leapfrogai_sdk.embeddings.embeddings_pb2_grpc import (
    EmbeddingsService,
    EmbeddingsServiceServicer,
    EmbeddingsServiceStub,
)
from leapfrogai_sdk.name.name_pb2 import NameResponse
from leapfrogai_sdk.name.name_pb2_grpc import (
    NameService,
    NameServiceServicer,
    NameServiceStub,
)
from leapfrogai_sdk.serve import serve

print("Initializing Leapfrog")
